tests/queries/0_stateless/03236_create_query_ttl_where.reference

```diff
@@ -0,0 +1 @@
+CREATE TABLE default.ttl\n(\n    `a` UInt32,\n    `timestamp` DateTime\n)\nENGINE = MergeTree\nORDER BY a\nTTL timestamp + toIntervalSecond(2) WHERE a IN (\n    SELECT number\n    FROM system.numbers\n    LIMIT 100000\n)\nSETTINGS index_granularity = 8192
```

Comment:
Previously it would have been
```
TTL timestamp + toIntervalSecond(2) WHERE a IN ((SELECT number FROM system.numbers LIMIT 100000) AS _subquery1)
```